deploy-manage/security/k8s-transport-settings.md

```diff
@@ -75,6 +75,12 @@ spec:
 
 ## Issue node transport certificates with third-party tools [k8s-transport-third-party-tools]
 
+:::{warning}
+Transport connections between {{es}} nodes are security-critical and you must protect them carefully. Malicious actors who can observe or interfere with node-to-node transport traffic can read or modify cluster data. A malicious actor who can establish a transport connection might be able to invoke system-internal APIs, including APIs that read or modify cluster data.
```

Comment:
This first paragraph might be better-placed at the top of the page, since the whole page is about securing transport connections.